web/src/common/theme.ts

```diff
@@ -1,10 +1,47 @@
 /**
  * @file Theme utilities.
  */
-import { UIConfig } from "@goauthentik/common/ui/config";
+import {
+    type StyleRoot,
+    createStyleSheetUnsafe,
+    setAdoptedStyleSheets,
+} from "@goauthentik/web/common/stylesheets.js";
+import { UIConfig } from "@goauthentik/web/common/ui/config.js";
+
+import AKBase from "@goauthentik/web/common/styles/authentik.css";
+import AKBaseDark from "@goauthentik/web/common/styles/theme-dark.css";
+import PFBase from "@patternfly/patternfly/patternfly-base.css";
 
 import { Config, CurrentBrand, UiThemeEnum } from "@goauthentik/api";
 
+//#region Stylesheet Exports
+
+/**
+ * A global style sheet for the Patternfly base styles.
+ *
+ * @remarks
+ *
+ * While a component *may* import its own instance of the PFBase style sheet,
+ * this instance ensures referential identity.
```

Comment:
This is most relevant when applying global styles to the document (via Storybook), or to  a web component (via Interface)